cmd/patch2pr/main.go

```diff
@@ -188,50 +224,37 @@ func execute(ctx context.Context, client *github.Client, patchFile string, opts
 		return nil, fmt.Errorf("get commit for %s failed: %w", patchBase, err)
 	}
 
-	var r io.ReadCloser
-	if patchFile == "-" {
-		r = os.Stdin
-	} else {
-		f, err := os.Open(patchFile)
+	var patches []Patch
```

Comment:
Since `parse` returns a `*Patch`, I'd probably make this a `[]*Patch` as well to avoid the dereference on L233 (or make `parse` return a value, using an empty `Patch` if there is an error.)

Reply:
Makes sense. I think using *Patch everywhere is the better option of the two.